examples/shadow_mapping.rs

```diff
@@ -5,15 +5,19 @@ extern crate cgmath;
 use cgmath::SquareMatrix;
 use glium::{glutin, Surface};
 use std::time::Instant;
+use glutin::dpi::LogicalSize;
 
 fn main() {
-    let win_size = (800, 600);
+    let win_size = LogicalSize {
+        width: 800.0,
+        height: 600.0,
+    };
     let shadow_map_size = 1024;
 
     // Create the main window
     let mut events_loop = glutin::EventsLoop::new();
     let window = glutin::WindowBuilder::new()
-        .with_dimensions(win_size.0, win_size.1)
+        .with_dimensions(win_size)
         .with_title("Shadow Mapping");
     let context = glutin::ContextBuilder::new().with_vsync(true);
```

Comment:
Nit: What you have here isn't a context, but a context builder. The name `context` would be slightly misleading. How about something short, like `cb`, or something not so short, like `context_builder`?

Reply:
Yes, but its not this PR which has added the code. Maybe something for a later PR that focuses on refactoring?